api/client/src/testFixtures/java/org/projectnessie/client/auth/oauth2/ResourceOwnerEmulator.java

```diff
@@ -213,11 +216,13 @@ private void triggerDeviceCodeFlow(URL initialUrl, String userCode) {
     try {
       LOGGER.info("Starting device code flow.");
       Set<String> cookies = new HashSet<>();
-      URL loginPageUrl = enterUserCode(initialUrl, userCode, cookies);
+      HttpURLConnection codeActionConn = enterUserCode(initialUrl, userCode, cookies);
```

Comment:
Should we close this connection when done?

Reply:
`HttpURLConnection` is not closeable. What "closes" the request is when a terminal action is invoked, e.g. `getResponseCode()`.

Reply:
:facepalm:

Reply:
`disconnect()` maybe?... although for lightly used test code it's probably fine to ignore.

Reply:
Frankly I don't think it will make any difference here, but why not.

Reply:
thx!